dist/gaia-header.js

```diff
@@ -514,6 +516,20 @@ function addDirObserver() {
   }
 }
 
+/**
+ * Copy the values of all properties from
+ * source object `target` to a target object `source`.
+ * It will return the target object.
+ *
+ * @param   {Object} target
+ * @param   {Object} source
+ * @returns {Object}
+ */
+function mixin(target, source) {
+  for (var key in source) target[key] = source[key];
```

Comment:
Because to properly polyfill `Object.assign()` means implementing the complete logic.
There is the [code for a complete polyfill](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign) in MDN, but I doubt we need something that complex here.